### PR TITLE
Update readme instructions

### PR DESCRIPTION
**Pull request by marrony**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,14 +20,43 @@ Currently, the driver is tested on:
 
 ### Installing
 
+To get the latest version use:
```
Comment (erickpintor) [inline README.md]: We should warn people about breaking changes.

File: README.md [hunk 2]
```diff
@@ -20,14 +20,46 @@ Currently, the driver is tested on:
...
```
Comment (ashfire908) [inline README.md]: `To get the latest version, run:`

File: README.md [hunk 3]
```diff
@@ -20,14 +20,46 @@ Currently, the driver is tested on:
...
 ```bash
 go get github.com/fauna/faunadb-go/faunadb
 ```
 
+Be aware that lately our driver undergo a breaking change so this method is not recommended anymore.
+If you use to install our driver using this method, please consider use one of the following tools.
+
+#### Using `gopkg.in`
+
+If you're using `gopkg.in`, to get a specific version use:
```
Comment (ashfire908) [inline README.md]:
```
To get a specific version when using `gopkg.in`, use:
```

File: README.md [hunk 4]
```diff
@@ -20,14 +20,46 @@ Currently, the driver is tested on:
...
+
+```bash
+go get gopkg.in/fauna/faunadb-go.v2/faunadb
+```
+
+#### Using `dep`
+
+If you're using `dep`, to get a specific version use:
```
Comment (ashfire908) [inline README.md]:
```
To get a specific version when using `dep`, use:
```

File: README.md [hunk 5]
```diff
@@ -20,14 +20,46 @@ Currently, the driver is tested on:
...
+```
+
 ### Importing
 
 For better usage, we recommend that you import this driver with an alias import
-such as:
+
+#### Using `gopkg.in`
+
+If you're using `gopkg.in` to import a specific version use:
```
Comment (ashfire908) [inline README.md]:
```
To import a specific version when using `gopkg.in`, use:
```

File: README.md [hunk 6]
```diff
@@ -20,14 +20,46 @@ Currently, the driver is tested on:
...
+
+```go
+import f "gopkg.in/fauna/faunadb-go.v2/faunadb"
+```
+
+#### Using `dep` or `go get`
+
+If you're using `dep` or `go get` import using:
```
Comment (ashfire908) [inline README.md]:
```
To import a specific version when using `dep` or `go get`, use:
```

File: README.md [hunk 7]
```diff
@@ -20,14 +20,46 @@ Currently, the driver is tested on:
 
 ### Installing
 
+To get the latest version use:
+
 ```bash
 go get github.com/fauna/faunadb-go/faunadb
 ```
 
+Be aware that lately our driver undergo a breaking change so this method is not recommended anymore.
```
Comment (ashfire908) [inline README.md]:
```
Please note that our driver undergoes breaking changes from time to time, so depending on our `master` branch is not recommended.
```

File: README.md [hunk 8]
```diff
@@ -20,14 +20,46 @@ Currently, the driver is tested on:
...
+If you use to install our driver using this method, please consider use one of the following tools.
```
Comment (ashfire908) [inline README.md]:
```
It is recommended to use one of the following methods instead.
```

File: README.md [hunk 9]
```diff
@@ -20,14 +20,46 @@ Currently, the driver is tested on:
...
 ### Installing
 
+To get the latest version run:
+
 ```bash
 go get github.com/fauna/faunadb-go/faunadb
 ```
 
+Please note that our driver undergoes breaking changes from time to time, so depending on our `master` branch is not recommended.
+It is recommended to use one of the following methods instead.
```
Comment (ashfire908) [inline README.md]: `.` -> `:` (my bad)

File: README.md [hunk 10]
```diff
@@ -20,14 +20,46 @@ Currently, the driver is tested on:
...
+
+To get a specific version when using `dep`, use:
+
+```bash
+dep ensure -add github.com/fauna/fauadb-go/faunadb@v2.0.0
+```
+
 ### Importing
 
 For better usage, we recommend that you import this driver with an alias import
```
Comment (ashfire908) [inline README.md]: End with period?

**Pull request merged**